app/presenters/publishing_api_presenters/edition.rb

```diff
@@ -21,6 +21,16 @@ def public_updated_at
     end
 
     def as_json
+      if edition.access_limited?
```

Comment:
After talking to Brad about the publishing API story, we agreed that even though we're stripping the access_limited data there, we should also ensure that in clients we control we don't send it for non-draft content. So we should add a check for `edition.draft?` here.